components/match2/wikis/osu/match_summary.lua

```diff
@@ -0,0 +1,330 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Abbreviation = require('Module:Abbreviation')
+local Class = require('Module:Class')
+local DateExt = require('Module:Date/Ext')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+
+local GREEN_CHECK = '<i class="fa fa-check forest-green-text" style="width: 14px; text-align: center" ></i>'
+local NO_CHECK = '[[File:NoCheck.png|link=]]'
+local MAP_VETO_START = '<b>Start Map Veto</b>'
+local ARROW_LEFT = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+local ARROW_RIGHT = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+local NONE = '-'
+local TBD = Abbreviation.make('TBD', 'To Be Determined') --[[@as string]]
+local ICONS = {
+	check = GREEN_CHECK,
+}
+
+local LINK_DATA = {
+	preview = {icon = 'File:Preview Icon32.png', text = 'Preview'},
+	mplink = {icon = 'File:Osu single color allmode.png', text = 'Match Data'},
+}
+	LINK_DATA.mplink2 = LINK_DATA.mplink
+	LINK_DATA.mplink3 = LINK_DATA.mplink
```

Comment:
What's the difference?

Reply:
There can be a case where it contains 3 different match data links for a single series

Reply:
Is it because it's 1 per game?

Reply:
> Is it because it's 1 per game?
No, one match data would cover the whole series (all maps), but from what I've been told theres a case where tech issue will resulted in them creating another lobby, therefore 1 full series can contain up to 3 of it (instead of usually just 1 for whole series)